etc/ci/performance/runner.py

```diff
@@ -237,19 +260,16 @@ def main():
         testcases = load_manifest(args.tp5_manifest)
         results = []
         for testcase in testcases:
-            command = (["timeout", "{}s".format(args.timeout)] +
-                       command_factory(testcase))
+            run_test = command_factory(testcase)
```

Comment:
Since we pass `testcase` down below to `run_test`, no need to pass it here. (Can probably also get rid of a layer of indirection/function calls.)